dashboard/test/models/unit_group_test.rb

```diff
@@ -194,6 +194,54 @@ class UpdateScriptsTests < ActiveSupport::TestCase
     assert_nil summary[:scripts][0]['stageDescriptions']
   end
 
+  test 'summarize with numbered units' do
+    unit_group = create :unit_group, name: 'my-unit-group'
+    script = create(:script, name: 'script1')
+    create(:unit_group_unit, unit_group: unit_group, position: 1, script: script)
```

Comment:
Can you add at least one other script so this test verifies that the numbering works correctly with multiple scripts? I'd be open to arguments that this is overkill, but in general I feel better testing with multiple elements in a list, if a list is involved.